### PR TITLE
R_shift had empty error coord and was causing problems for assigning the equilibrium to plasma

### PR DESCRIPTION
**Pull request by michael-gemmell**
R_shift had empty error coord and was causing problems for assigning the equilibrium to plasma

### CODE REVIEW

File: indica/readers/abstractreader.py [hunk 1]
```diff
@@ -219,14 +219,20 @@ def get_ppts(
                 dims = ["t"]
             else:
                 raise ValueError(f"Unknown quantity: {quantity}")
+            if quantity == "R_shift":
```
Comment (marcosertoli) [inline indica/readers/abstractreader.py:222]: I don't understand where this is throwing an error. In assign_dataarray, if there is not ".._error" quantity, then a default error = xr.zeros_like(data) is assigned. 

I haven't had any issues with this reader and have included it in both the Zeff and the Prad workflow.

If possible, I would sort this out where you get the error and leave this reader method as it was.

**Review by marcosertoli — COMMENTED** [review]
I would fix where you get the error and revert to reader method as in the main branch.

File: indica/readers/abstractreader.py (hunk 1 above)
Comment (michael-gemmell) [reply indica/readers/abstractreader.py:222]: The issue is that the R_shift gets a default error and then that propogates to the equilibrium which then gives me some error when I try plasma.set_equilibrium

Comment (marcosertoli) [reply indica/readers/abstractreader.py:222]: What error? 
I'd rather find out what's going wrong, eventually changing the R_shift assignment in the Equilibrium if you needed.

**Review by marcosertoli — CHANGES_REQUESTED** [review]

**Pull request closed without merging**